lib/diagnostic.rb

```diff
@@ -106,10 +106,10 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = puts "#{person[:first_name]} #{person[:age]}"
```

Comment:
This will cause an error since `person` doesn't exist.  The answer that we're looking for is a statement that creates a hash called `person` with the requested keys and values.  `person = { first_name: 'really good name', age: 12345 }`.